docs/cookbook/dynamic-configuration/01_Basics/03_assets/final.yml

```diff
@@ -0,0 +1,253 @@
+server:
```

Comment:
Can we manually trim out zero values from the final.yml files so it's not overwhelming to readers? (even if that means it's not synced with the actual output of /-/config anymore)

Reply:
I struggled with this, partly due to laziness and partly since I wanted to match exactly what agentctl dryrun outputs. I tried to call out what should be in the final. I may see if I can add an option to dryrun to compare a default config with the generated and only output the differences.

Reply:
👍 might be a good opportunity for us to add missing `omitempty` all over the place (though it doesn't help with all our dependencies that don't have it)